### DISCUSSION TITLE
Sort snapshot keys

### DISCUSSION THREAD

**Discussion by keesvanbemmel** [directus/directus#9188] [Feature Requests]
Hi there,

Could we please sort the snapshot YAML when we create a snapshot? This makes comparing changes much easier.

I'd suggest changing this line: [https://github.com/directus/directus/blob/ac0bade2b4f382ba696840a31019c00c3b73f954/api/src/cli/commands/schema/snapshot.ts#L47](https://github.com/directus/directus/blob/ac0bade2b4f382ba696840a31019c00c3b73f954/api/src/cli/commands/schema/snapshot.ts#L47)

Into:
`await fs.writeFile(filename, toYaml(snapshot, {'sortKeys': true}));`

**Comment by joselcvarela** [comment]
Hello @keesvanbemmel 
Do you mind opening a PR?
I think this is a nice feature to have.

**Reply by keesvanbemmel** [reply]
https://github.com/directus/directus/pull/9198

**Comment by keesvanbemmel** [comment]
Sure! Just wanted to discuss first as described in contributing guidelines
:)

**Comment by licitdev** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
